etc/ATSCam/segment_hdu.header

```diff
@@ -12,3 +12,29 @@ DTM1_1  =                      / detector transformation matrix
 DTM2_2  =                      / detector transformation matrix
 DTM1_2  =                      / detector transformation matrix
 DTM2_1  =                      / detector transformation matrix
+WCSNAMEA= 'AMPLIFIER'          / Name of coordinate system
+CTYPE1A = 'Seg_X'              / In the camera coordinate system
+CTYPE2A = 'Seg_Y'              / In the camera coordinate system
+PC1_1A  =
+PC1_2A  =
+PC2_1A  =
+PC2_2A  =
+CDELT1A =
+CDELT2A =
+CRPIX1A =
+CRPIX2A =
+CRVAL1A =
+CRVAL2A =
+WCSNAMEC= 'CCD'                / Name of coordinate system
+CTYPE1C = 'CCD_X'              / In the camera coordinate system
+CTYPE2C = 'CCD_Y'              / In the camera coordinate system
+PC1_1C  =
+PC1_2C  =
+PC2_1C  =
```

Comment:
I don't think we should be writing per-segment WCS information.  It's implied by the whole-CCD Wcs, and the information required to go from amp to CCD geometry is in `cameraGeom` so should not be repeated here.

Reply:
There is a niche case, but running LSST code easily assembles the segments even without running full ISR, and has utilities to give the (ra, dec) for a position in amp coordinates.  I think that the danger of all this extra information embedded in the header service outweighs the advantages

Reply:
I am agnostic in regards to adding the `PCX_XXC` matrix. However Tony made the request to include these keywords as the Camera software uses it for visualization and that's why I added in the spirit of making the files useful for most people. Please tell me how you would like me to proceed.

Reply:
This has been removed now.